spec/rails_helper.rb

```diff
@@ -77,6 +80,7 @@
   end
 
   config.before(:each, geocode: true) do
+    allow(Geocoder).to receive(:search).and_call_original
```

Comment:
I needed to unstub the search method in order for the geocode tests to run as expected!